instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './SearchBar.css';
+
+
+const SearchBar = props => {
+    return(
+        <div className="search-bar">
+        <div className="instagram">
+        <img className="icon" src="https://img.icons8.com/ios/50/000000/instagram-new.png" alt="instagram icon"/>
+        <img className ="insta-logo" src="instagram-logo.png" alt="instagram name logo"/>
```

Comment:
Using your src like this won't work you would need to import the image and use that in your src